spec/models/piece_spec.rb

```diff
@@ -1,5 +1,42 @@
 require 'rails_helper'
 
 RSpec.describe Piece, type: :model do
-  
-end
+  describe "#move_to!" do
+    context "when another piece is on the new spot" do
+      it "captures it by updating its coords to nil" do
+        piece1 = Piece.create(x_cord: 0, y_cord: 0, color: "white")
```

Comment:
ahh, okay I kind of am getting the point of this, and kind of not.  Give me another keyword help here... I think at first I thought we would be able to establish current game using params :game_id in the controller later?